src/store/content.js

```diff
@@ -0,0 +1,57 @@
+export const actions = {
+	async getResources({
+		commit
+	}, payload = {}) {
+		commit("setLoadingTrue")
```

Comment:
```suggestion
		commit("setLoading", true)
```
You could write a generic method (mutation) for that, that accepts a parameter
https://vuex.vuejs.org/guide/mutations.html#commit-with-payload